modules/tensor_mechanics/src/materials/ComputeExtraStressConstant.C

```diff
@@ -0,0 +1,31 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#include "ComputeExtraStressConstant.h"
+
+template<>
+InputParameters validParams<ComputeExtraStressConstant>()
+{
+  InputParameters params = validParams<ComputeExtraStressBase>();
+  params.addClassDescription("Computes a constant extra stress that is added to the stress calculated by the constitutive model");
+  params.addRequiredParam<std::vector<Real> >("extra_stress_tensor", "Vector of values defining the constant extra stress to add, in order 11, 22, 33, 23, 13, 12");
+  params.addParam<MaterialPropertyName>("prefactor", 1.0, "Name of material defining additional constant prefactor or variable dependence");
```

Comment:
I would not explicitly mention Variable dependence, because you do not provide any derivatives to the stress.